src/Hadolint/Formatter/Format.hs

```diff
@@ -13,7 +13,6 @@ module Hadolint.Formatter.Format
 import Data.List (sort)
 import qualified Data.List.NonEmpty as NE
 import Data.Monoid (Monoid)
-import Data.Semigroup
```

Comment:
This is required for compiling hadolint with olde GHC versions

Reply:
Ok I will put it back